Workflow/Sources/WorkflowHost.swift

```diff
@@ -49,10 +49,18 @@ public final class WorkflowHost<WorkflowType: Workflow> {
     /// - Parameter workflow: The root workflow in the hierarchy
     /// - Parameter debugger: An optional debugger. If provided, the host will notify the debugger of updates
     ///                       to the workflow hierarchy as state transitions occur.
-    public init(workflow: WorkflowType, debugger: WorkflowDebugger? = nil) {
+    public init(
+        workflow: WorkflowType,
+        observers: [WorkflowObserver]? = nil,
```

Comment:
Do `[]` and `nil` have different meaning as values for the `observers` parameter?

Reply:
no – i can remove the optionality